mfs/ops.go

```diff
@@ -100,9 +100,9 @@ func PutNode(r *Root, path string, nd *dag.Node) error {
 
 // Mkdir creates a directory at 'path' under the directory 'd', creating
 // intermediary directories as needed if 'mkparents' is set to true
-func Mkdir(r *Root, pth string, mkparents bool, flush bool) error {
+func Mkdir(r *Root, pth string, mkparents bool, flush bool) (*Directory, error) {
```

Comment:
You're right: this was left over from an earlier approach I took. It seems nice to have (giving back the dir you asked me to create), but I can nix it if you'd rather not keep it around.